lib/js/styles/ItemsTypography.vue

```diff
@@ -43,6 +44,18 @@ export default {
 				result += item.property + ': var(--typography-' + item.value + ');';
 			});
 
+			console.log(result);
+			return result;
+		},
+		variableToValue(variable) {
+			let result: string = '';
+			(Object.keys(variables) as Array<keyof typeof variables>).forEach((key) => {
```

Comment:
why you don't iterate over `Object.values()`?

Reply:
Fixed.